doc/admin-guide/files/records.config.en.rst

```diff
@@ -1313,6 +1313,8 @@ HTTP Redirection
    This setting determines the maximum number of times Trafficserver does a redirect follow location on receiving a 3XX Redirect response
    for a given client request.
 
+   .. note:: When :ts:cv:`proxy.config.http.number_of_redirections` is set to a positive value and |TS| has previously cached a 3XX Redirect response, the cached response will continue to be refreshed and returned until the response is no longer in the cache.
+
```

Comment:
![screen shot 2018-09-25 at 15 37 06](https://user-images.githubusercontent.com/905298/46041647-e6a76880-c0d8-11e8-8bfa-d4ac0ef08c31.png)

Rendering at 906c6df7b5a37f8068a9579c48e8cbc42147fc3a.

Reply:
@bryancall Does this look OK to you?